tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/ClearDatabase.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using NHSD.BuyingCatalogue.Identity.Api.Data;
+using TechTalk.SpecFlow;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests
+{
+    [Binding]
+    public sealed class ClearDatabase
+    {
+        [AfterScenario]
+        public void Clear(IConfigurationRoot config)
+        {
+            var optionsBuilder = new DbContextOptionsBuilder<ApplicationDbContext>();
```

Comment:
These should move into another method in the database class, which should be called from here, so the non-builder DB code is in one place.